From 3b1b28acd3c30402911e1571d64638fee57cba7c Mon Sep 17 00:00:00 2001
From: Gregory Haskins <ghaskins@novell.com>
Date: Tue, 25 Aug 2009 20:31:19 -0400
Subject: [PATCH 025/119] venettap: disable burstthresh by default

This feature causes performance regressions in more cases than it helps
so lets make the default value "off".

Signed-off-by: Gregory Haskins <ghaskins@novell.com>
---
 kernel/vbus/devices/venet-tap.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/kernel/vbus/devices/venet-tap.c b/kernel/vbus/devices/venet-tap.c
index d6c36ca..8a67624 100644
--- a/kernel/vbus/devices/venet-tap.c
+++ b/kernel/vbus/devices/venet-tap.c
@@ -1646,7 +1646,7 @@ venettap_device_create(struct vbus_devclass *dc,
 
 	priv->vbus.rx_ops      = &venettap_flat_rx_ops;
 	init_waitqueue_head(&priv->vbus.rx_empty);
-	priv->burst.thresh     = 20; /* microseconds */
+	priv->burst.thresh     = 0; /* microseconds, 0 = disabled */
 
 	/*
 	 * netif init
-- 
1.6.5.2

